Dockerfile

```diff
@@ -1,13 +1,11 @@
 FROM openjdk:8-jre
 
-COPY build/install/spring-boot-template /opt/app/
+COPY build/install/send-letter-producer-consumer /opt/app/
```

Comment:
*consumer

Reply:
oh it's `producer-consumer`. should it be just `consumer`?

Reply:
omg

Reply:
fixed